server/src/analytics/GoogleAnalytics.ts

```diff
@@ -38,11 +39,7 @@ export class GoogleAnalytics implements Analytics {
 
   public async sendPageView(taskName: string): Promise<void> {
     try {
-      if (
-        this.serverState?.env !== "production" ||
```

Comment:
Do we not want to block ga in dev?

Reply:
I removed this condition while testing and then thought it may be ok to allow analytics on development, but I can roll back that change